src/connectors/toggle-refinement/connectToggleRefinement.js

```diff
@@ -218,7 +218,10 @@ export default function connectToggleRefinement(renderFn, unmountFn = noop) {
       getRenderState(renderState, renderOptions) {
         return {
           ...renderState,
-          toggleRefinement: this.getWidgetRenderState(renderOptions),
+          toggleRefinement: {
+            ...renderState.toggleRefinement,
+            [attribute]: this.getWidgetRenderState(renderOptions),
+          },
```

Comment:
uiState change is technically a breaking change, but small enough to just mention in changelog